keyboards/the_ruler/readme.md

```diff
@@ -0,0 +1,14 @@
+The Ruler PCB
+=======
+
+A custom keyboard PCB ruler, that can also function as a macro pad
+
+Keyboard Maintainer: Maple Computing
+Hardware Supported: PCB Ruler V1 by That-Canadian and Bishop Keyboards
+Hardware Availability: https://www.maple-computing.com/products/pcb-ruler-v1-1
+
+Make example for this keyboard (after setting up your build environment):
+
+    make the_ruler-default
```

Comment:
Would you mind updating this? `make the_ruler:default` is the new format :)

Reply:
done